### ISSUE TITLE
Create Retina-savvy toolbar and application icons

### ISSUE THREAD

**Issue by splorp** [pguyot/Einstein#17]
Yeah, those icons are soooo 2006.

A couple of things …

I’m happy to start tackling this, but was wondering if @pguyot (or anyone else) knows who created the original icons. It would be handy to be able to supersize those images by starting with some source files.

Also, are there any other images that need updating besides those found in the `Resources` folder?

https://github.com/pguyot/Einstein/tree/master/Resources/icons

**Comment by jake-b** [comment]
iOS Builds have a .xib launch screen which is required to let the OS know that the application supports certain screen sizes and rotations.   Right now its really boring.  We could put a logo or something on there.

One of my working builds had something I hand drew on my iPad just to have a place holder:
https://github.com/pguyot/Einstein/blob/ObjCBridging/_Build_/Xcode6/logo.png

... but @panicsteve is the artist here. 👍

**Comment by splorp** [comment]
Ha! That’s hilarious.

Part of me would love to do a series of chunky bitmap riffs on the original Newton OS icons.

**Comment by splorp** [comment]
Hmmm … I wonder if we could convince the folks at Iconfactory to do something for us?

**Comment by pguyot** [comment]
The original icons were created by Michael Vacík (http://linkedin.com/in/mvacik)

**Comment by splorp** [comment]
Thanks, Paul. I had a hunch that it was Michael, since he did the _Powered by NPDS_ button back in the day. I’ll reach out to him and see if he still has any of the source files.

![serve-npds](https://cloud.githubusercontent.com/assets/29835/22172871/6b486d3e-df6f-11e6-8cfd-b28cdfbe2d0b.gif)

**Comment by MatthiasWM** [comment]
I added the ugly "network_in" icon at some point to symbolize a network PCMCIA card was inserted into the machine. Feel free to scale and/or replace ;-)

**Comment by splorp** [comment]
Getting back to one of my original questions … do all of the pertinent images reside in the following folder or are their other locations?

https://github.com/pguyot/Einstein/tree/master/Resources/icons

**Comment by MatthiasWM** [comment]
Yes, I think those are the only MacOS images. There are two png's here: ./app/iEinstein/ , but they are iOS icons AFAIK. All other png's are for Android, or for the documentation

**Comment by MatthiasWM** [comment]
By now, we have a few more images in ```Resources/Icons``` to support multiple PCMCIA cards. But still just a hand full.

**Comment by MatthiasWM** [comment]
Is there any progress here? Should I leave this open until someone has the time to scale the images?
